### PR TITLE
Move all tests in a single directory

### PR DESCRIPTION
**Pull request by wizmer**

### CODE REVIEW

**Comment by wizmer** [comment]
Let's merge this one first

File: tests/test_io.py [hunk 1]
```diff
@@ -88,10 +88,27 @@ def test_load_population():
...
+
+
+def test_tree_type():
+    tree_types = {5: 'soma',
+                  6: 'axon',
+                  7: 'basal',
+                  8: 'apical'}
+
+    neuron = io.load_neuron(os.path.join(DATA_PATH, 'basic_exotic_section_types.swc'),
+                               soma_type=5,
```
Comment (wizmer) [inline tests/test_io.py]: It's a bit confusing that the soma type is defined both by the tree_types dictionary and the soma_type argument

File: tmd/io/io.py [hunk 2]
```diff
@@ -54,8 +54,9 @@ def load_neuron(input_file, line_delimiter='\n', soma_type=None,
     that there are disconnected components
     '''
 
+    tree_types_final = td.copy()
     if tree_types is not None:
-        td.update(tree_types)
```
Comment (wizmer) [inline tmd/io/io.py]: This is a serious bug. Each time `load_neuron` is called with the `tree_types` argument it modifies the td dictionary. And the modification will persist between each call of `load_neuron`
You might have loaded wrong section types with realizing it.

Comment (lidakanari) [reply tmd/io/io.py]: Ouch, good catch! The objective was to modify the tree_types for the lifetime of the load_neuron. Thanks for fixing this one!

**Comment by wizmer** [comment]
I have activated the 'review required' so you need to give to explicitly give your approval on the PR now.

File: tests/data/basic_exotic_section_types.swc [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Comment line
+1 5 0 0 0 6 -1
+3 8 3 4 5 6 1
```
Comment (lidakanari) [inline tests/data/basic_exotic_section_types.swc]: If the first row is ID it cannot be duplicated, so it should be 1,2,3... Unless this file is used to catch an error.

**Comment by lidakanari** [comment]
Should I merge this one?

File: tests/data/basic_exotic_section_types.swc (hunk 3 above)
Comment (mgeplf) [reply tests/data/basic_exotic_section_types.swc]: Doesn't this mean that the SWC reader is doing something strange?

Comment (wizmer) [reply tests/data/basic_exotic_section_types.swc]: It is still using the custom reader ?

Comment (lidakanari) [reply tests/data/basic_exotic_section_types.swc]: Sorry forgot this issue. What do you mean? I have an swc reader for TMD so if that's what you mean yes. This file is still wrong though (you cannot have two lines with the same ID in an swc file).

**Pull request merged**